pest/examples/parens.rs

```diff
@@ -0,0 +1,105 @@
+extern crate pest;
```

Comment:
Is this an example of what the generated code will be like? If so, some of my comments about this may not apply.